.github/workflows/build.yml

```
@@ -119,14 +119,16 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        arch: ["arm64", "amd64"]
+        arch: ["arm64", "amd64", "s390x"]
     env:
       repo: ${{github.event.repository.name}}
       version: ${{needs.get-product-version.outputs.product-version}}
 
     steps:
       - uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
-      - name: Docker Build (Action)
+      - name: Docker Build (Action) ${{ matrix.arch }}
+        # linux/s390x images can only ever be published the Red Hat catalog. In the future they may be pushed to ICR as well.
+        if: "${{ matrix.arch != 's390x'}}"
```

IIRC this doesn't need the outer quotes.

```suggestion
        if: ${{ matrix.arch != 's390x'}}
```

---

updated in 3d1660a2eac2078ae9c4fe60fb9bcff83bacd775